styles/lobste.rs/catppuccin.user.css

```diff
@@ -0,0 +1,153 @@
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    background-color: @base;
+
+    --palette-bg: @base;
+    --palette-fg: @text;
+
+    --color-fg: @text;
+    --color-fg-shape: @surface2;
+
+    --color-tag-bg: fade(@yellow, 20%);
+    --color-tag-border: fade(@yellow, 20%);
+
+    --color-lobsters-tag-special-bg: fade(@red, 30%);
+    --color-lobsters-tag-special-border: fade(@red, 30%);
+
+    --color-tag-media-bg: fade(@blue, 20%);
+    --color-tag-media-border: fade(@blue, 30%);
+
+    --color-fg-affirmative: @green;
+    --color-fg-accent: @accent-color;
+    --color-fg-author: @blue;
+
+    --color-fg-contrast-7-5: @blue;
+    --color-fg-contrast-4-5: @subtext0;
+    --color-fg-contrast-6: @subtext1;
+
+    --color-fg-link: @blue;
+    --color-fg-link-visited: @lavender;
+
+    --color-fg-contrast-10: @text;
+
+    --color-box-bg: @mantle;
+    --color-button-bg: @mantle;
+    --color-button-bg-shaded: @surface0;
+    --color-box-border: @surface0;
+    --color-box-border-focus: @accent-color;
+    --color-box-bg-shaded: @mantle;
+
+    --color-mobile-story-liner-bg: @base;
+    --color-mobile-story-comments-bubble-fill: @surface1;
+
+    --color-mobile-story-comments-bubble-fill-zero: @surface0;
+
+    --color-table-header-bg: @surface0;
+    --color-table-header-border: @surface1;
+    --color-table-row-bg-even: @mantle;
+    --color-table-row-bg-odd: @crust;
+    --color-table-row-border: @base;
+
+    --color-flash-bg-success: fade(@green, 30%);
+
+    #logo {
+      @svg: e(
+        escape(
+          '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><g><path fill="@{accent-color}" d="M16,16c-5.33,0-10.67,0-16,0C0,10.67,0,5.33,0,0c5.33,0,10.67,0,16,0C16,5.33,16,10.67,16,16z"/><path fill="@{base}" d="M3.5,13.45c0-0.1,0-0.2,0-0.3c0-0.11,0-0.22,0-0.33c0-0.11,0-0.1,0.1-0.11c0.23-0.02,0.46-0.03,0.69-0.07   c0.46-0.08,0.67-0.31,0.74-0.78c0.02-0.14,0.03-0.29,0.03-0.43c0-2.25,0-4.5,0-6.74c0-0.18-0.02-0.36-0.04-0.53   C4.99,3.85,4.79,3.66,4.5,3.55c-0.27-0.1-0.55-0.11-0.83-0.13c-0.05,0-0.1,0-0.15,0c0-0.03,0-0.06,0-0.08c0-0.19,0-0.38,0-0.57   c0-0.07,0.02-0.09,0.09-0.09c1.04,0,2.07,0,3.11,0c0.71,0,1.42,0,2.13,0c0.07,0,0.1,0.01,0.1,0.09c-0.01,0.19,0,0.37,0,0.56   c0,0.03,0,0.05,0,0.08c-0.14,0.01-0.28,0-0.41,0.02c-0.2,0.03-0.4,0.06-0.59,0.11C7.6,3.65,7.42,3.89,7.38,4.22   c-0.02,0.15-0.03,0.3-0.03,0.45c0,2.13,0,4.26,0,6.39c0,0.27,0.03,0.54,0.06,0.8c0.03,0.29,0.22,0.48,0.49,0.58   c0.23,0.09,0.48,0.11,0.72,0.12c0.63,0.02,1.25,0.01,1.86-0.15c0.88-0.22,1.47-0.77,1.76-1.64c0.09-0.28,0.13-0.57,0.2-0.86   c0.01-0.02,0.03-0.06,0.05-0.06c0.23,0,0.42,0,0.66,0c-0.02,1.2,0.02,2.4,0,3.6C9.94,13.45,6.71,13.45,3.5,13.45z"/></g></svg>'
+        )
+      );
```

Comment:
You shouldn't need to wrap this in the `e` function.

Reply:
Huh, when I tried it earlier, if I didn't wrap it with `e` LESS would throw an error along the lines of "invalid % without a number", but it's not doing that anymore. I'll remove it.

Reply:
Right, when I was testing I hadn't replaced the colors yet. Anyway, I removed it now.